pkg/internal/vault/vault.go

```diff
@@ -383,13 +383,19 @@ func (v *Vault) IsVaultInitializedAndUnsealed() error {
 	healthURL := path.Join("/v1", "sys", "health")
 	healthRequest := v.client.NewRequest("GET", healthURL)
 	healthResp, err := v.client.RawRequest(healthRequest)
+	if err != nil {
+		return fmt.Errorf("error calling Vault %s: %w", healthURL, err)
+	}
+
+	defer healthResp.Body.Close()
```

Comment:
And I guess we can safely assume that wheneve the error is _not_ nil there will always be body 👍🏼

Reply:
Yep :+1: https://github.com/hashicorp/vault/blob/ac56e5571dfe0c4166bf72def6045632da2d776b/api/client.go#L952